app/controllers/events/view/export.js

```diff
@@ -0,0 +1,53 @@
+import Ember from 'ember';
+
+const { Controller, run } = Ember;
+export default Controller.extend({
+  eventExportStatus  : '',
+  eventDownloadUrl   : '',
+  isDownloadDisabled : true,
+  image              : false,
+  audio              : false,
+  video              : false,
+  document           : false,
+  actions            : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('image'),
+        video    : this.get('video'),
+        audio    : this.get('audio'),
+        document : this.get('document')
+      };
+      this.get('loader')
+        .post(`/events/${this.get('model.id')}/export/json`, payload)
+        .then(exportJobInfo => {
+          let requestLoop = run.later(() => {
```

Comment:
@niranjan94 I have kept it in such a way that if the server response is WAITING, then it will keep running every 3 seconds else if the server returns direct failure I am stopping the loop. Also on SUCCESS response, I am terminating the loop.